tokio/src/sync/mpsc/bounded.rs

```diff
@@ -332,6 +333,7 @@ impl<T> Sender<T> {
     ///
     /// [`close`]: Receiver::close
     /// [`Receiver`]: Receiver
+    /// [`PollSender`]: https://docs.rs/tokio-util/0.6.4/tokio_util/sync/struct.PollSender.html
```

Comment:
```suggestion
    /// [`PollSender`]: https://docs.rs/tokio-util/0.6/tokio_util/sync/struct.PollSender.html
```

Reply:
Why don't we just put /latest/ here instead of concrete version?

Reply:
The link may break if we move it or remove it in future versions of `tokio-util`.

Reply:
Good to know, thanks!